components/FeaturedPost.tsx

```diff
@@ -15,28 +18,21 @@ const FeaturedPost = () => {
               height="32"
               alt="author"
             />
-            <span className="text-sm text-left text-gray-700 group-hover:text-primary-600">Simon Al Kamina</span>
+            <span className="text-sm text-left text-gray-700 group-hover:text-primary-600">{author}</span>
           </div>
         </a>
-        <p className="mt-4 leading-relaxed font-serif text-gray-500">
-          Embedded programming is always different with regular high-level programming with humanlike APIs--they tend to
-          be closer to the metal, and less easy for humans to read.
-        </p>
+        <p className="mt-4 leading-relaxed font-serif text-gray-500">{desc}</p>
         <Link href="/blog">
           <a className="bg-primary-900 text-white py-1.5 px-10 inline-block mt-4">READ MORE</a>
         </Link>
       </div>
       <div className="w-5/6 lg:w-full mx-auto lg:mx-0">
         <div className="aspect-w-16 aspect-h-13 w-full">
-          <img
-            className="w-full object-cover object-center"
-            src="https://picsum.photos/500"
-            alt="Modern C++ on Arduino"
-          />
+          <img className="w-full object-cover object-center" src="https://picsum.photos/500" alt={title} />
```

Comment:
related to previous comment, imagenya juga. tapi seingetku belom ada frontmatter tag buat cover imagenya. ini jadi notes aja, siapa tau mau support kedepannya.